Minor changes to readme (#136)
COSIMA/regional-mom6: README.md

Limitations: Currently the package supports only one horizontal grid type (that is equally spaced in longitude); there are plans to add more grid options. We have designed the package in a way that it is modular so, for example, one needs to implement just another method for a different type of grid and the rest should be good to go.

If you find this package useful and have any suggestions please feel free to open an [issue](https://github.com/COSIMA/regional-mom6/issues) or a [discussion](https://github.com/COSIMA/regional-mom6/discussions). We'd love to have [new contributors](https://regional-mom6.readthedocs.io/en/latest/contributing.html) and we are very keen to help you out along the way!

## What you need to get started:
1. a cool idea for a new regional MOM6 domain,
2. a working MOM6 executable on a machine of your choice, 
3. a bathymetry file that at least covers your domain,

> 4. 3D ocean forcing files *of any resolution* on your choice of A, B or C Arakawa grid,

Reviewer: ```suggestion
4. 3D ocean forcing files *of any resolution* on your choice of A, B, or C Arakawa grid,
```

> 5. surface forcing files (eg ERA or JRA reanalysis), and

Reviewer: ```suggestion
5. surface forcing files (e.g., from ERA or JRA reanalysis), and
```

> 6. [GFDL's FRE tools](https://github.com/NOAA-GFDL/FRE-NCtools) must be downloaded and compiled on the machine you are using.

Reviewer: ```suggestion
6. [GFDL's FRE tools](https://github.com/NOAA-GFDL/FRE-NCtools) be downloaded and compiled on the machine you are using.
```
